tests/unittest/test_helpers.py

```diff
@@ -232,11 +232,11 @@ def side_effect(*args, **kwargs):
 
         label = helpers.utils.get_primary_label()
 
-        cmd_get_service_account = f"{test_id}/kubectl --kubeconfig {kubeconfig} --namespace {namespace} --context {context}  get serviceaccount -l {label} -A -o yaml"
+        cmd_get_service_account = f"{test_id}/opt/spark/kubectl --kubeconfig {kubeconfig} --namespace {namespace} --context {context}  get serviceaccount -l {label} -A -o yaml"
```

Comment:
maybe it is worth factoring out the kubectl location into a single variables, just to have this in a single place

Reply:
:+1: 

Will push the changes once kubectl location is confirmed.